package.json

```diff
@@ -13,6 +13,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "aws-sdk": "^2.249.1",
```

Comment:
This shouldn't be a dependency -- otherwise, aws-sdk will be imported twice, as s3-stream-upload doesn't pull it in itself, but use it as an argument

Reply:
removing. Ideally this project would transparently load and use the aws-sdk dependency.